analyzers/tests/SonarAnalyzer.UnitTest/Rules/ControlCharacterInStringTest.cs

```diff
@@ -25,19 +25,28 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ControlCharacterInStringTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<ControlCharacterInString>();
+
         [TestMethod]
         public void ControlCharacterInString() =>
```

Comment:
NITPICK: rename the test to ControlCharacterInString_CSharp8. As it is applicable only from chsarp8.